.github/workflows/ci.yml

```
@@ -1,54 +1,79 @@
       - name: Install dependencies
         run: uv sync
-      
+
       - name: Run ruff check
-        run: uv run ruff check
+
+  test:
+    name: Test Suite
+    runs-on: ubuntu-latest
+
+    steps:
```

actionlint (1.7.11)

[error] could not parse as YAML: did not find expected '-' indicator

(syntax-check)